File: 본문/java/Code_2_02_2.java
import java.util.*;

class Code_2_02_2 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt(); // a 입력
		int b = sc.nextInt(); // b 입력
		System.out.println(a & b); // a AND b의 값을 출력
		System.out.println(a | b); // a OR b의 값을 출력
		System.out.println(a ^ b); // a XOR b의 값을 출력
	}
};

File: 본문/c/Code_2_02_2.c
#include <stdio.h>

int main() {
	int a, b;
	scanf("%d%d", &a, &b); // a와 b를 입력
	printf("%d\n", a & b); // a AND b의 값을 출력
	printf("%d\n", a | b); // a OR b의 값을 출력
	printf("%d\n", a ^ b); // a XOR b의 값을 출력
	return 0;
}
